### PR TITLE
[TD-11626]<feature>schemaless use sample from random

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

**Review by afwerar — APPROVED** [review]
LGTM

File: src/benchData.c [hunk 1]
```diff
@@ -649,18 +651,34 @@ int prepareSampleDataWithStb(SSuperTable *stbInfo) {
...
+                           char *data_type, int32_t *data_length, int64_t size,
+                           uint16_t iface) {
     for (int64_t i = 0; i < size; i++) {
         int32_t pos = i * lenOfOneRow;
         for (int c = 0; c < count; c++) {
             char *tmp = NULL;
             switch (data_type[c]) {
                 case TSDB_DATA_TYPE_BINARY:
                 case TSDB_DATA_TYPE_NCHAR: {
+                    if (data_length[c] > TSDB_MAX_BINARY_LEN) {
```
Comment (sangshuduo) [inline src/benchData.c:663]: why not avoid user pass over-large length from command line or JSON parse?

**Review by sangshuduo — COMMENTED** [review]
see my comments

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**